collector/serverstats.go

```diff
@@ -145,7 +205,7 @@ func (e Exporter) getServerstatsMetrics(logger log.Logger, ch chan<- prometheus.
 	}
 	level.Debug(logger).Log("msg", "Got 'serverstats' response", "serverstats_packet", packet.GetStatus())
 
-	serverstats, ok := packet.(*chrony.ReplyServerStats3)
+	serverstats, ok := packet.(*chrony.ReplyServerStats4)
```

Comment:
Hmm, I wonder if we need to do some checks to find out what packet type is really coming back.

Reply:
> Hmm, I wonder if we need to do some checks to find out what packet type is really coming back.

Of course we should. I'll try to find time in the coming days to address this (also including the even older `ReplyServerStats` and `ReplyServerStats2`).

Reply:
Any luck finding time to work on this?